cli/command/container/hijack.go

```diff
@@ -19,41 +19,38 @@ import (
 // TODO: This could be moved to `pkg/term`.
 var defaultEscapeKeys = []byte{16, 17}
 
-// A hijackedIOStreamer handles copying input to and output from streams to the
+// HijackOptions handles copying input to and output from streams to the
 // connection.
-type hijackedIOStreamer struct {
-	streams      command.Streams
-	inputStream  io.ReadCloser
-	outputStream io.Writer
-	errorStream  io.Writer
-
-	resp types.HijackedResponse
-
-	tty        bool
-	detachKeys string
+type HijackOptions struct {
+	Resp         types.HijackedResponse
```

Comment:
Maybe `Hijacked` instead of `Resp` for the field name?

Reply:
I don't hold strong opinion on naming this guy. Changed to `Hijacked`